### PR TITLE
Add widget meta-data - v0.11.0

### PR DESCRIPTION
**Pull request by anxolin**
Ad widget meta-data

The idea is to be able to identify both the host app and the widget app in the onchain transaction.

The host app already adds its `appCode` in the root of the appData.
This PR adds a new metadata so we can also add the widget.

This might look silly, but serves these purposes:

- Makes it easier to know if an app uses the widget or not. Because the widget creates the appData, it can ensure this metadata is present. If its not, the only way you have to know for sure is to investigate, try to find the app and check its implementation. This way we know for sure, and is automatically accounted
- The other is, because i would expect others host the widget web themselves, or use IPFS versions of it. This way, the host app will not be CoW Swap, but it will be something else

```json
{
      "appCode": "Cow Protocol: Demo",
      "environment": "prod",
       "metadata": {
  "version": "0.11.0",
  "metadata": {
    "widget": {
      "appCode": "Cow Swap",
      "environment": "prod"
    }
  }
}
...
}

```

Adds tests:
image

### CODE REVIEW

**Review by shoom3301 — APPROVED** [review]

File: src/generatedTypes/index.ts [hunk 1]
```diff
@@ -11,18 +12,19 @@ import * as v0_7_0 from './v0.7.0'
...
+export * as latest from './v0.11.0'
 
-export const LATEST_APP_DATA_VERSION = '0.10.0'
+export const LATEST_APP_DATA_VERSION = '0.11.0'
 export const LATEST_QUOTE_METADATA_VERSION = '0.3.0'
 export const LATEST_REFERRER_METADATA_VERSION = '0.2.0'
 export const LATEST_ORDER_CLASS_METADATA_VERSION = '0.3.0'
 export const LATEST_UTM_METADATA_VERSION = '0.2.0'
 export const LATEST_HOOKS_METADATA_VERSION = '0.1.0'
 export const LATEST_SIGNER_METADATA_VERSION = '0.1.0'
```
Comment (alfetopito) [inline src/generatedTypes/index.ts:23]: Need to edit the compile script to have the export for the widget metadata version:

https://github.com/cowprotocol/app-data/blob/3c7aeb1a03132087ff22001a6e0cb2ad33dde1aa/src/scripts/compile.ts#L65

**Review by alfetopito — COMMENTED** [review]
Needs an update to the compile script

File: src/generatedTypes/index.ts (hunk 1 above)
Comment (anxolin) [reply src/generatedTypes/index.ts:23]: Done!

**Review by alfetopito — APPROVED** [review]

**Review by zaidusmanali — APPROVED** [review]

**Review by zaidusmanali — APPROVED** [review]
Run

**Pull request merged**